docs/framework-integration/angular.md

```diff
@@ -294,6 +294,21 @@ of your Angular workspace (`/packages/angular-workspace`), run the following com
 npx -p @angular/cli ng build component-library
 ```
 
+:::note
+In the output of the `ng build` command you may see a warning that looks like this: 
+
+```
+▲ [WARNING] The glob pattern import("./**/.entry.js") did not match any files [empty-glob]
+
+node_modules/@stencil/core/internal/client/index.js:3808:2:
+  3808 │   `./${bundleId}.entry.js${BUILD.hotModuleReplacement && hmrVers...
+       ╵   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+```
+
+This is a known issue in esbuild (used under the hood by `ng build`) and should
+not cause an issue.
```

Comment:
Saying "This is a known issue" _probably_ covers it, but it may be worthwhile to add a sentence saying we can't do anything about it (suppressing it or otherwise).

 ```suggestion
not cause an issue. At this time, this comment cannot be suppressed.
```

I'm happy with either, just something to consider. Non-blocking